src/llmcompressor/transformers/finetune/text_generation.py

```
@@ -63,179 +43,3 @@ def train(**kwargs):
     from llmcompressor import train
 
     train(**kwargs)
-
-
-def apply(**kwargs):
-    """
-    CLI entrypoint for any of training, oneshot
-    """
-    from llmcompressor.args import parse_args
-
-    model_args, dataset_args, recipe_args, training_args, _ = parse_args(
-        include_training_args=True, **kwargs
-    )
-
-    training_args.run_stages = True
-    report_to = kwargs.get("report_to", None)
-    if report_to is None:  # user didn't specify any reporters
-        # get rid of the reporters inferred from hugging face
-        training_args.report_to = []
-    main(model_args, dataset_args, recipe_args, training_args)
-
-
-def compress(**kwargs):
```

Please raise a deprecation error?

---

ok